### DISCUSSION TITLE
Is there any adobe creative cloud app (arm version) for apple silicon available on homebrew?

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#5671] [Casks] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

The one available there asks for Rosetta 2 so it is probably intended for x86-64 architecture. Is there anyway I can get the arm version?

### Relevant casks

adobe-creative-cloud

**Comment by daeho-ro** [comment]
It has arm version but caveats added. I will remove it.

**Reply by daeho-ro** [reply, marked as answer]
- https://github.com/Homebrew/homebrew-cask/pull/188432

**Reply by ghost** [reply]
Thanks for the reply.

> It has arm version but caveats added. I will remove it.

I installed it 2–3 days ago and seen that will require Rosetta 2. However, I didn't try to run and ended up removing that app.
